examples/lit-element-kitchen-sink/.storybook/webpack.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = async ({ config }) => {
```

Comment:
This is the place where help is needed. Please [see](https://github.com/Polymer/lit-element/issues/54#issuecomment-439901575)